src/py/flwr/server/state/state_test.py

```diff
@@ -341,7 +385,10 @@ def test_num_task_res(self) -> None:
 
 
 def create_task_ins(
-    consumer_node_id: int, anonymous: bool, delivered_at: str = ""
+    consumer_node_id: int,
+    anonymous: bool,
+    delivered_at: str = "",
+    workload_id: str = "",
```

Comment:
It might be safer to remove the defaults here and force callers that relied on the default to manually pass `""`.